homeassistant/components/arve/entity.py

```diff
@@ -0,0 +1,43 @@
+"""Arve base entity."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import ArveCoordinator
+
+
+class ArveDeviceEntity(CoordinatorEntity[ArveCoordinator]):
+    """Defines a base Arve device entity."""
+
+    _attr_has_entity_name = True
+    _attr_available = True
+
+    def __init__(self, coordinator: ArveCoordinator, description) -> None:
```

Comment:
```suggestion
    def __init__(self, coordinator: ArveCoordinator, description: EntityDescription) -> None:
```

`from homeassistant.helpers.entity import EntityDescription`